vendor/github.com/mattn/go-shellwords/shellwords_test.go

```diff
@@ -0,0 +1,132 @@
+package shellwords
```

Comment:
@SvenDowideit trash only includes files that are referenced (so not all .go) and no _test.go